docs/medical-api/api-reference/document/post-upload-url.mdx

```diff
@@ -0,0 +1,145 @@
+---
+title: "Upload Document"
+description: "Returns a URL to use for a medical document upload to our servers."
+api: "POST /medical/v1/document/upload"
+---
+
+This endpoint returns a URL to enable you to upload your patients' medical documents, making them available to other HIEs.
+
+## Overview
+
+To use this endpoint, you will need to follow these steps:
+
+1. Create a `DocumentReference` to describe the document that's being uploaded (see sections below for details on the expected fields and format).
+
+2. Execute the endpoint and receive a URL to use for the document upload.
+
+3. Upload the document by executing a `PUT` request using the `URL` with a `Content-Length` header specifying the size of the document.
+
+## Query Params
+
+<ParamField query="patientId" type="string" required>
+  The ID of the Patient.
+</ParamField>
+
+## Body
+
+    A FHIR [DocumentReference](/medical-api/fhir/resources/documentreference).
+    <ParamField body="type" type="CodeableConcept" required>
+      A [CodeableConcept](/medical-api/fhir/data-types/codeableconcept) of the document type.
+      <Expandable title="Type properties">
+        <ParamField body="text" type="string" required>
+          Plain text representation of the document kind - for example `Burn management Hospital Progress note`
+        </ParamField>
+
+        <ParamField body="coding" type="Coding[]">
+          A [Coding](/medical-api/fhir/data-types/coding) property, which is a reference to a code defined by a terminology system (such as [LOINC](https://loinc.org/)) - for example `{ code: "100556-0", system: "http://loinc.org" },`
```

Comment:
nit: the coding usually has a "display" which is a useful plain english description, right?

let's encourage the devs to provide as high-quality input as possible by example, in this case "
Burn management Hospital Progress note"